lib/browser/api/browser-actions.js

```diff
@@ -62,6 +64,8 @@ module.exports = {
   },
 
   getPopup: (extensionId, details) => {
-    return getBrowserAction(extensionId, details).popup
+    const action1 = getBrowserAction(extensionId, details)
+    const action2 = getBrowserAction(extensionId, {})
```

Comment:
@jonathansampson updated per your feedback 😄  Makes the change look a little smaller too